api/src/main/java/io/opencensus/stats/AggregationData.java

```diff
@@ -288,8 +293,52 @@ public static DistributionData create(
         Utils.checkNotNull(bucket, "bucket should not be null.");
       }
 
+      Utils.checkNotNull(exemplars, "exemplar list should not be null.");
+      Utils.checkArgument(
+          exemplars.size() <= bucketCounts.size(),
+          "each histogram bucket can have up to one exemplar, "
+              + "so the length of exemplar list should be no greater than that of bucket list.");
```

Comment:
I think we should either allow any number of exemplars or perform this check by comparing the exemplar values with the bucket boundaries.  The current check isn't complete, because it would allow two exemplars in one bucket if there were zero exemplars in another bucket.